source/_integrations/blink.markdown

```diff
@@ -140,6 +110,15 @@ homeassistant:
         - '/path/to/whitelist'
 ```
 
+### `blink.send_pin`
+
+Send a new pin to blink.  Since Blink's 2FA implementation is new and changing, this is to allow the integration to continue to work with user intervention.  The intent is to handle all of this behind the scenes, but until the login implementation is settled this was added.  To use it, you simply call the service with the pin you receive from Blink as the payload (for a simple "Allow this Device" email, you may keep the `pin` value empty).
+
+| Service Data Attribute | Optional | Description                  |
+| ---------------------- | -------- | ---------------------------- |
+| `pin`                  | no       | 2FA Pin received from blink. |
+
+
```

Comment:
```suggestion
### `blink.send_pin`

Send a new pin to blink.  Since Blink's 2FA implementation is new and changing, this is to allow the integration to continue to work with user intervention.  The intent is to handle all of this behind the scenes, but until the login implementation is settled this was added.  To use it, you simply call the service with the pin you receive from Blink as the payload (for a simple "Allow this Device" email, you may keep the `pin` value empty).

| Service Data Attribute | Optional | Description                  |
| ---------------------- | -------- | ---------------------------- |
| `pin`                  | no       | 2FA Pin received from blink. |

```